.github/CODEOWNERS

```diff
@@ -0,0 +1,10 @@
+# This file creates automatic review requests for PRs based on the files touched.
+# See this blog post for more info: https://github.com/blog/2392-introducing-code-owners
+
+# Later rules override earlier rules
+
+# As a fallback, ping the team
+* @mozilla/normandy-comitters
```

Comment:
I'd suggest making a `normandy-owners` team or something with just core committers (our immediate team, probably) rather than using `normandy-committers`, which includes QA and ops and will notify them when normally we don't have them doing reviews.